packages/devtools_app/test/split_test.dart

```diff
@@ -620,12 +620,14 @@ void main() {
             child: SizedBox(width: 400.0, height: 400.0, child: split),
           ),
         ));
-        expectEqualSizes(tester.element(find.byKey(_k1)).size,
-            const Size(182.5242718446602, 400));
-        expectEqualSizes(tester.element(find.byKey(_k2)).size,
-            const Size(10.951456310679607, 400));
-        expectEqualSizes(tester.element(find.byKey(_k3)).size,
-            const Size(182.5242718446602, 400));
+        // TODO(dantup): Why did these change? Which care correct? These match
+        // the original widths given a 400x400 SizedBox.
+        expectEqualSizes(
+            tester.element(find.byKey(_k1)).size, const Size(100.0, 400));
+        expectEqualSizes(
+            tester.element(find.byKey(_k2)).size, const Size(176.0, 400));
+        expectEqualSizes(
+            tester.element(find.byKey(_k3)).size, const Size(100.0, 400));
```

Comment:
Need to understand why these changed (to my eyes the new ones seem reasonable, but I'm not sure why before this change we had the others).

Reply:
Jacob explained this - when the box was resized down to 200, the proportions changed (out of line with what was originally set), and when it was re-increased, it should have maintained this new proportions. This is somehow being lost after my change so I'll revert these and investigate why.